ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/gcc_10/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+##########
+# gcc_10 #
+##########
+
+- name: Check if gcc 10.3 is installed on RHEL/CentOS/Ubuntu16
```

Comment:
As per [Ansible docs](https://docs.ansible.com/ansible/2.8/user_guide/playbooks_error_handling.html#overriding-the-changed-result), this task will report "changed" whenever run (meaning it's changed the machine state), because it's the `shell` module. As this is just a querying task, `changed_when: False` should be used, as then it will only report `ok`

Reply:
As per above I think this will be the same for the other GCC roles and probably others. A good candidate for a separate issue/PR I feel

Reply:
Raised at https://github.com/adoptium/infrastructure/issues/2422 and since I'm having to run extra VPCs anyway I've added that into this PR too now - thanks.

Reply:
Sure, I can do that quickly enough, at least for the other GCC roles. Thanks for putting it in this PR though :-)